# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/moz.build
# Commit: d7f55df2c2e4
# Full Hash: d7f55df2c2e4305928afec2b632f72517ccfb35f
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-29 15:31:45
# Regressor Bug: 1831451
# File Overlap Count: 2
# Description:
#   Bug 1831451 - Guess color space from VideoData r=aosmond
#   
#   Depends on D171940
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D170635
# ==============================================================================

diff -r 61be9a6b9cd4 -r d7f55df2c2e4 dom/media/webcodecs/moz.build
--- a/dom/media/webcodecs/moz.build	Thu Jun 29 04:15:03 2023 +0000
+++ b/dom/media/webcodecs/moz.build	Thu Jun 29 04:15:03 2023 +0000
@@ -33,4 +33,8 @@
     "WebCodecsUtils.cpp",
 ]
 
+if CONFIG["MOZ_WAYLAND"]:
+    CXXFLAGS += CONFIG["MOZ_WAYLAND_CFLAGS"]
+    CFLAGS += CONFIG["MOZ_WAYLAND_CFLAGS"]
+
 FINAL_LIBRARY = "xul"
